examples/worlds/breadcrumbs.sdf

```diff
@@ -464,6 +464,7 @@
       </plugin>
       <plugin filename="libignition-gazebo-breadcrumbs-system.so" name="ignition::gazebo::systems::Breadcrumbs">
        <topic>/B2/deploy</topic>
+       <disable_physics_time>2.0</disable_physics_time>
```

Comment:
Just curious, is this different from sim time?

Reply:
the calculation is based on sim time. So in this case if the breadcrumb is spawned at sim time = 3.0s, it will become static at sim time = 5.0s

Reply:
Oh, I see. I took it to mean "disable" after "physics time", not "disable physics" after "time"

Reply:
yeah I think I've changed the name of this param 3 times. Naming is hard. Feel free to suggest better names :)